lib/helpers/git_helper.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+require 'git'
+require 'fileutils'
+
+def get_config (config_file)
```

Comment:
This should be in a Application level helper file, not associated with a controller. Touched on this in PR #8